presto/dbapi.py

```diff
@@ -298,13 +299,27 @@ def _format_prepared_param(self, param):
         Formats parameters to be passed in an
         EXECUTE statement.
         """
+        if isinstance(param, bool):
+            return "true" if param else "false"
+        if isinstance(param, float):
+            return "DOUBLE '%s'" % param
         if isinstance(param, str):
             return ("'%s'" % param.replace("'", "''"))
-        else:
-            # TODO: param could have many other types which should
-            #       be covered including unicode, maps, lists, etc.
-            #       https://github.com/prestosql/presto-python-client/issues/45
-            return str(param)
+        if isinstance(param, bytes):
+            return "X'%s'" % param.hex()
```

Comment:
can you please add support for unicode as well?
for now this can be as simple as the `str` case.
in the future we can go with `U&'...'` unicode escapes, but maybe we do not actually need to

Reply:
did some reading, in python 3 all `str` objects are unicode, thus should we always apply the `U&` prefix? also, any suggestions for doing unicode detection that is compatible with both  python2 and python3?

Reply:
```
isinstance(param, type(u''))
```

or, perhaps more elegant way
```
try:
    from builtins import unicode
except ImportError:
    unicode = str
```

Reply:
> should we always apply the U& prefix

let's not use U& for now. just handling `unicode` _type_ is enough for this PR.

as a follow up we can add some unicode test cases (with ascii, non-ascii, non-BMP, etc.)

Reply:
OK, let's keep unicode out of this PR. https://github.com/prestosql/presto-python-client/issues/53